src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/RealWorldContextTests.cs

```diff
@@ -218,11 +219,11 @@ public virtual void RoundtripWithCustomConverterProperty_Struct()
         [Fact]
         public virtual void RoundTripWithCustomPropertyConverterFactory_Class()
         {
-            const string Json = "{\"MyEnum\":\"A\"}";
+            const string Json = "{\"MyEnum\":\"One\"}";
 
             ClassWithCustomConverterPropertyFactory obj = new()
             {
-                MyEnum = SampleEnum.A
+                MyEnum = SampleEnum.One
```

Comment:
Why the enum value change here?

Reply:
I meant to comment on this. There are two "SampleEnum" types and there were collisions once some `using` statements were added for other reasons, so I removed the enum type that was recently added.